miasm2/ir/translators/z3_ir.py

```diff
@@ -0,0 +1,308 @@
+import logging
+import operator
+
+import z3
+
+import miasm2.expression.expression as m2_expr
+from miasm2.ir.translators.translator import Translator
+
+log = logging.getLogger("translator_z3")
+console_handler = logging.StreamHandler()
+console_handler.setFormatter(logging.Formatter("%(levelname)-5s: %(message)s"))
+log.addHandler(console_handler)
+log.setLevel(logging.WARNING)
+
+class Z3Mem(object):
```

Comment:
Thanks for your details !

Actually, I misunderstood the purpose of the argument `size`. But your docstring `@size: integer, size in bit of addresses in the memory to get.` is enough; I didn't see it in the first commit.

So, the current implementation sounds good for me.